### PR TITLE
Merge master branch into stable-2.361 for LTS 2.361.2 release

### PR DESCRIPTION
**Pull request by krisstern**
Prepare for LTS release of 2.361.2

### CODE REVIEW

**Comment by NotMyFault** [comment]
Is any of that change actually needed for the next point release?

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
> Is any of that change actually needed for the next point release?

yes it looks like at least the nuget changes are

File: systemd/jenkins.service [hunk 1]
```diff
@@ -84,18 +84,12 @@ Environment="JENKINS_PORT=@@PORT@@"
...
-# See the Winstone documentation for more information.
 #Environment="JENKINS_HTTP2_LISTEN_ADDRESS="
 
 # HTTP2 port to listen on. Default is disabled.
 # To be able to listen on privileged ports (port numbers less than 1024),
 # add the CAP_NET_BIND_SERVICE capability to the AmbientCapabilities
 # directive below.
-#
-# Note: HTTP2 support may require additional configuration.
-# See the Winstone documentation for more information.
```
Comment (timja) [inline systemd/jenkins.service:98]: I don't think these docs changes should be backported, it still needs additional config in LTS as far as I can tell

**Comment by NotMyFault** [comment]
> > Is any of that change actually needed for the next point release?
> 
> 
> 
> yes it looks like at least the nuget changes are

I backported the change before.

**Comment by krisstern** [comment]
I am thinking maybe I should open another PR to do the backporting other than dumping the entire `master` branch to the `stable-2.361` branch, since I don't think I can cherry pick the commits this way.

**Comment by krisstern** [comment]
Just wanted to close this PR because I have checked the commit bc929b3c634282eb148ebbdc069b33f8a57a3964 has indeed been backported so nothing further is needed to be done here.

**Pull request closed without merging**